python/gpusim_search.py

```diff
@@ -9,19 +10,27 @@ def main():
 
     socket = QtNetwork.QLocalSocket(app)
     smiles = input("Smiles: ")
-    db_name = sys.argv[1]
-    socket.connectToServer(db_name)
+    dbcount = 1
+    dbname = sys.argv[1]
+    dbkey = sys.argv[2]
```

Comment:
Is it worth adding a simple argparser to ensure it's called the right way?

Reply:
Yeah, will do